simple_app/app_simple.py

```diff
@@ -210,11 +270,78 @@ def solo_result(query: str):
     query: str
         The query -- full match to a main ID
     """
+    curdoc().template_variables['query'] = query  # add query to bokeh curdoc
     db = SimpleDB(db_file, connection_arguments={'check_same_thread': False})  # open database
     resultdict: dict = db.inventory(query)  # get everything about that object
     query = query.upper()  # convert query to all upper case
     everything = Inventory(resultdict)  # parsing the inventory into markdown
-    return render_template('solo_result.html', query=query, resultdict=resultdict, everything=everything)
+    return render_template('solo_result.html', resources=CDN,
+                           query=query, resultdict=resultdict, everything=everything)
+
+
+@app_simple.route('/camdplot')
+def camdplot():
+    """
+    Creates CAMD plot as JSON object
+
+    Returns
+    -------
+    plot
+        JSON object for page to use
+    """
+    db = SimpleDB(db_file, connection_arguments={'check_same_thread': False})  # open database
+    query: str = curdoc().template_variables['query']  # get the query (on page)
+    if args.debug:
+        print(query)
+        print(all_photo)
+    resultdict: dict = db.inventory(query)  # get everything about that object
+    everything = Inventory(resultdict)  # parsing the inventory
+    # TODO: hover to show object name, ref, relevant colour, be clickable to trigger query
```

Comment:
Maybe write an issue to handle the updates to hover and a separate one to handle clicks.

Reply:
An old example of this: https://github.com/dr-rodriguez/AstrodbWeb/blob/2e1210822fb85c47ff6db286843a0ff4b829f5b7/bdnyc_app/app.py#L620

Reply:
I've pushed that functionality live now